plugins/source/postgresql/resources/plugin/plugin_test.go

```diff
@@ -76,33 +135,105 @@ func createTestTable(ctx context.Context, conn *pgxpool.Pool, table *schema.Tabl
 	return nil
 }
 
-func insertTestTable(ctx context.Context, conn *pgxpool.Pool, table *schema.Table, data schema.CQTypes) error {
-	var sb strings.Builder
-	sb.WriteString("INSERT INTO ")
-	sb.WriteString(table.Name)
-	sb.WriteString(" (")
-	for i, col := range table.Columns {
-		sb.WriteString(col.Name)
-		if i < len(table.Columns)-1 {
-			sb.WriteString(", ")
+func insertTestTable(ctx context.Context, conn *pgxpool.Pool, tableName string, columns []pgTypeToValue) error {
+	var query = ""
+	query += "INSERT INTO " + tableName + " ("
+	for _, col := range columns {
+		if col.value == nil {
+			continue
 		}
+		query += pgx.Identifier{col.typeName + "_type"}.Sanitize() + ", "
```

Comment:
clash on columns with the same type

Reply:
See previous column

Reply:
Ah, I reviewed and didn't notice that this is a test.